examples/seismic/model.py

```diff
@@ -345,17 +345,23 @@ class Model(object):
     """
 
     def __init__(self, origin, spacing, shape, space_order, vp, nbpml=20,
-                 dtype=np.float32, epsilon=None, delta=None, theta=None, phi=None):
+                 dtype=np.float32, epsilon=None, delta=None, theta=None, phi=None,
+                 **kwargs):
         self.shape = shape
         self.nbpml = int(nbpml)
         self.origin = tuple([dtype(o) for o in origin])
 
         shape_pml = np.array(shape) + 2 * self.nbpml
         # Physical extent is calculated per cell, so shape - 1
         extent = tuple(np.array(spacing) * (shape_pml - 1))
-        self.grid = Grid(extent=extent, shape=shape_pml,
-                         origin=origin, dtype=dtype)
-
+        # Check for input grid
+        self.grid = kwargs.get('grid', None)
```

Comment:
the pythonic way is:
```
self.grid = kwargs.get('grid', Grid(extent=extent, shape=shape_pml, origin=origin, dtype=dtype))
```

Reply:
No, because some if `grid=None` in `kwargs` then `self.grid` is None